src/lightkurve/get_skycatalog.py

```diff
@@ -0,0 +1,179 @@
+"""
+THIS IS A WORK IN PROGRESS
+
+This tool will provide the user with a list of stars/objects within a defined region around a requested target.
+
+The returned list contains the proper motion corrected R.A and Dec. postions of the objects in addition to several other parameters.
+
+This tool can be applied to the following TESS products
+- LightCurve Objects
+- TargetPixelFile Objects
+- TESSCut Objects
+
+Eventual use case will be:
+mytpf.get_skycatalog()
+Returning a table
+
+
+This file will likely be merged into the search.py code and structure 
+It is placed here as an initial working document 
+"""
+
+import numpy as np
+import astropy.units as u
+from astropy.time import Time
+from astroquery.mast import Catalogs
+from astropy.coordinates import SkyCoord
+from astroquery.vizier import Vizier
+from astropy.table import Table
+
+
+# This creates a dictionary of the relevant catalogs in VizieR
+_catalog_dict = {
+    "kepler": "V/133",
+    "k2": "IV/34",
+    "tess": "IV/39/tic82",
+    "gaia": "I/355",
+}
+
+
+# This function should take the information from the skycatalog search and apply the proper motion
+def apply_propermotion(catalog: Table, equinox: Time, epoch: Time):
+    """
+    Function that returns an astropy table of sources with the proper motion applied
+
+    Parameters:
+    -----------
+    catalog :
+        astropy.table.Table which contains the coordinates of targets and proper motion values
+    equinox: astropy.time.Time
+        The R.A and Dec. vaulues taken from the catalogs is in J2000.
+        The J2000. 0 epoch is precisely Julian date 2451545.0 TT.
+    epoch : astropy.time.Time
+        Time of the observation - This is taken from the catalog R.A and Dec. values and re-formatted as an astropy.time.Time object
+
+    Output:
+    ------
+    catalog : astropy.table.Table
+        Returns an astropy table with ID, corrected RA, corrected Dec, and Mag(?Some ppl might find this benifical for contamination reasons?)
+    """
+
+    # Get the input data from the catalog
+    c = SkyCoord(
+        catalog["RAJ2000"],
+        catalog["DEJ2000"],
+        pm_ra_cosdec=catalog["pmRA"],
+        pm_dec=catalog["pmDEC"],
+        frame="icrs",
+        obstime=equinox,
+    )
+
+    c1 = c.apply_space_motion(new_obstime=epoch)
+
+    catalog["RAJ2000"] = c1.ra.to(u.deg).value
+    catalog["DEJ2000"] = c1.dec.to(u.deg).value
+
+    return catalog
+
+
+def query_skycatalog(
+    coord: SkyCoord,
+    epoch: Time,
+    catalog_name: str,
+    radius: float = 20.0,
```

Comment:
You could accept an astropy quantity here
```suggestion
    radius: Union[float, u.Quantity] = u.Quantity(20, 'arcsecond'),
```
And then in the code you would ensure radius has units of arcsecond
```python
try: 
    radius = u.Quantity(radius, 'arcsecond')
except u.UnitConversionError:
    raise
```

Reply:
Great suggestion! 
Thanks